subsystem/kubeapi_test.go

```diff
@@ -1636,7 +1636,7 @@ var _ = Describe("[kube-api]cluster installation", func() {
 				agent.Status.NtpSources[0].SourceName == common.TestNTPSourceSynced.SourceName &&
 				agent.Status.NtpSources[0].SourceState == models.SourceStateUnreachable
 
-		}, "30s", "10s").Should(BeTrue())
+		}, "30s", "1s").Should(BeTrue())
```

Comment:
sorry i probably wasn't clear, lets have both 60s timeout and 1s interval.

Reply:
done:)